hr_timesheet_task_domain/models/account_analytic_line.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016 Tecnativa - Sergio Teruel
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    task_id = fields.Many2one(
```

Comment:
I'm not sure this is working correctly. In the image, the domain appears different. This project has all tasks in a folded stage, so these tasks should not be shown.

![image](https://github.com/user-attachments/assets/68f563ce-a9df-43fd-bb6d-d12a6cf4fead)

![image](https://github.com/user-attachments/assets/f49fb8ba-221a-4af2-b448-35adb31aab17)

Reply:
On the other hand, Odoo currently considers a task closed based on the `state=1_done` field rather than on whether the `stage is folded`. I think it might be good to adapt to this approach. What do you think?

![image](https://github.com/user-attachments/assets/1ddace7e-2a45-4853-b44e-36348d1acc5e)